src/Components/Youtube/Youtube.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Youtube extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            
+        }
+    }
+
+    render() {
+        return (
+            <div className='video-responsive'>
+                <iframe
+                    width='654'
+                    height='380'
+                    src={`https://www.youtube.com/embed/${this.props.embededId}`}
+                    frameBorder='0'
+                    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                    allowFullScreen
+                    title="Embedded youtube"
+                  />
```

Comment:
Looks great! Need to do some more styling. I wonder if we can add a className to iframe and set our styling in there